test/ClangImporter/objc_bridging.swift

```diff
@@ -57,15 +59,44 @@ func mutablePointerToObjC(_ path: String) throws -> NSString {
 
 func objcStructs(_ s: StructOfNSStrings, sb: StructOfBlocks) {
   // Struct fields must not be bridged.
-  _ = s.nsstr! as Bool // expected-error {{cannot convert value of type 'Unmanaged<NSString>' to type 'Bool' in coercion}}
-
+  _ = s.nsstr! as Bool // expected-error {{cannot convert value of type 'NSString' to type 'Bool' in coercion}}
+  
   // FIXME: Blocks should also be Unmanaged.
-  _ = sb.block as Bool // expected-error {{cannot convert value of type '@convention(block) () -> Void' to type 'Bool' in coercion}}
+  _ = sb.block as Bool // expected-error {{cannot convert value of type '() -> Void' to type 'Bool' in coercion}}
```

Comment:
I never understood if this change is "for the better" or we are introducing a unintentional difference.

It might be worth it to mention on the Source Compatibility section of the pitch. I think it does not have any practical effects, but changing the convention from `block` to `swift` might break some code.

Reply:
Ah yeah I'll revisit this